<ui version="4.0" >
 <class>GameView</class>
 <widget class="QWidget" name="GameView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="_3" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible" >
      <bool>false</bool>
     </property>
     <widget class="QWidget" native="1" name="ui_leftBox" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="_4" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" native="1" name="ui_boardBox" >
         <property name="minimumSize" >
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" native="1" name="ui_buttonBox" >
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="ui_flip" >
            <property name="text" >
             <string>flip</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ui_begin" >
            <property name="text" >
             <string>begin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ui_backward" >
            <property name="text" >
             <string>backward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ui_forward" >
            <property name="text" >
             <string>forward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ui_end" >
            <property name="text" >
             <string>end</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" native="1" name="ui_rightBox" >
      <layout class="QVBoxLayout" name="_2" >
       <item>
        <widget class="QGroupBox" name="ui_infoBox" >
         <property name="title" >
          <string>Info:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ui_playersBox" >
         <property name="title" >
          <string>Players:</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" native="1" name="ui_playersWidget" >
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="ui_whitePlayersFrame" >
               <property name="frameShape" >
                <enum>QFrame::Panel</enum>
               </property>
               <property name="frameShadow" >
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" >
                <property name="spacing" >
                 <number>0</number>
                </property>
                <property name="margin" >
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" >
                  <property name="spacing" >
                   <number>0</number>
                  </property>
                  <item>
                   <spacer>
                    <property name="orientation" >
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType" >
                     <enum>QSizePolicy::MinimumExpanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0" >
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="ui_whitePlayerIcon" >
                    <property name="text" >
                     <string/>
                    </property>
                    <property name="alignment" >
                     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                    <property name="margin" >
                     <number>2</number>
                    </property>
                    <property name="indent" >
                     <number>2</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="ui_whitePlayer" >
                    <property name="text" >
                     <string/>
                    </property>
                    <property name="alignment" >
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="margin" >
                     <number>2</number>
                    </property>
                    <property name="indent" >
                     <number>2</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer>
                    <property name="orientation" >
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType" >
                     <enum>QSizePolicy::MinimumExpanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0" >
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="ui_whiteClock" >
                  <property name="font" >
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="frameShape" >
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="text" >
                   <string/>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="margin" >
                   <number>2</number>
                  </property>
                  <property name="indent" >
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="ui_blackPlayersFrame" >
               <property name="frameShape" >
                <enum>QFrame::Panel</enum>
               </property>
               <property name="frameShadow" >
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" >
                <property name="spacing" >
                 <number>0</number>
                </property>
                <property name="margin" >
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" >
                  <property name="spacing" >
                   <number>0</number>
                  </property>
                  <item>
                   <spacer>
                    <property name="orientation" >
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType" >
                     <enum>QSizePolicy::MinimumExpanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0" >
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="ui_blackPlayerIcon" >
                    <property name="text" >
                     <string/>
                    </property>
                    <property name="alignment" >
                     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                    <property name="margin" >
                     <number>2</number>
                    </property>
                    <property name="indent" >
                     <number>2</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="ui_blackPlayer" >
                    <property name="text" >
                     <string/>
                    </property>
                    <property name="alignment" >
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="margin" >
                     <number>2</number>
                    </property>
                    <property name="indent" >
                     <number>2</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer>
                    <property name="orientation" >
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType" >
                     <enum>QSizePolicy::MinimumExpanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0" >
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="ui_blackClock" >
                  <property name="font" >
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="frameShape" >
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="text" >
                   <string/>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="margin" >
                   <number>2</number>
                  </property>
                  <property name="indent" >
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ui_piecesBox" >
         <property name="title" >
          <string>Pieces:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ui_movesBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Moves:</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QTabWidget" name="ui_tabWidget" >
            <property name="currentIndex" >
             <number>0</number>
            </property>
            <widget class="QWidget" name="ui_moveTable" >
             <property name="geometry" >
              <rect>
               <x>0</x>
               <y>0</y>
               <width>133</width>
               <height>492</height>
              </rect>
             </property>
             <attribute name="title" >
              <string>Table</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="ui_moveInline" >
             <property name="geometry" >
              <rect>
               <x>0</x>
               <y>0</y>
               <width>133</width>
               <height>490</height>
              </rect>
             </property>
             <attribute name="title" >
              <string>Notation</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
